### PR TITLE
fix linux flann build

### PR DESCRIPTION
**Pull request by svensht2**
//cc @DillonSkeehan

### CODE REVIEW

File: src/cpp/CMakeLists.txt [hunk 1]
```diff
@@ -29,7 +29,7 @@ if (BUILD_CUDA_LIB)
 endif()
 
 if(CMAKE_SYSTEM_NAME STREQUAL "Linux" AND CMAKE_COMPILER_IS_GNUCC)
-    add_library(flann_cpp SHARED "")
+    add_library(flann_cpp SHARED "flann/empty.cpp")
```
Comment (DillonSkeehan) [inline src/cpp/CMakeLists.txt:32]: how /why does this fix the cmake error?

Comment (svensht2) [reply src/cpp/CMakeLists.txt:32]: it doesn't like the empty string, so I gave it a dummy file

Comment (DillonSkeehan) [reply src/cpp/CMakeLists.txt:32]: guess i'm a little confused about why there was an add library call that is empty originally. seems like that would never work? I wonder if they were just using this a dummy library?

Comment (svensht2) [reply src/cpp/CMakeLists.txt:32]: they were linking a static library into it, don't know why

**Review by DillonSkeehan — APPROVED** [review]

**Pull request merged**